bin/logs.js

```diff
@@ -32,6 +32,12 @@ module.exports = Cli.createCommand('logs', 'Streaming, real-time logs', {
             'default': 'default',
             type: 'string',
         },
+        lines: {
```

Comment:
I think I would prefer something like `history` (but not sure if that even communicates properly). To me, `lines` suggest that this is **all** you will get.